data/remnant missions.txt

```diff
@@ -493,5 +598,761 @@ mission "Remnant: Technology Available"
+					decline
+				"I needed a vacation anyway."
+				accept
+	on complete
+		conversation
+			`	Once you've landed on <planet> you look up the local university and pay it a visit. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`
+
+mission "Remnant: Follow-up on Terminus Researchers 3"
+	name "Check up on Terminus Researchers"
+	description "You offered to checkup on the wormhole researchers"
+	source
+		government Deep
+	destination
+		government Remnant
+	to offer
+		has "Remnant: Follow-up on Terminus Researchers 2: done"
+	on offer
+		conversation
+			`Back on board the <ship>, you take a moment to flip through the papers.  You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
+			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: They have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`
+			`	Prefect Reegar is definitely going to be interested in seeing this.`
+			accept
+	on complete
+		payment 2000000
+		conversation
+			`The trip back to the Ember Wastes is long, and gives you plenty of opportunity to think about what this research could lead to. You also wonder what the Remnant will do, as it seems obvious that their current isolation is about to fail.`
+			`	Reegar seems grimly optimistic on receiving your report "If this had happened without us being aware, it could have gone badly. Fortunately for us, you were in the right place at the right time, and have proven yourself loyal to us. So we are now forewarned and can use this time to plan and prepare for it. Perhaps re-establishing contact with the rest of humanity could even be to our benefit, but that will be for the council to decide."`
+			`	"Thank you, Captain. If you ever come across any other interesting information like this, don't hesitate to bring it to me."`
+
+mission "Remnant: Busy Stranger"
+	landing
+	invisible
+	to offer
+		has "Remnant: Return the Samples 2: done"
+		has "Remnant: Salvage 6: done"
+		has "Remnant: Expanded Horizons Quarg 3: done"
+	source
+		government "Remnant"
+	on offer
+		payment 2000000
+		log "The prefects have suggested that you've done all the jobs currently available, but they are workign on more."
```

Comment:
"workign" to "working". I suggest to change the log to "The prefects have suggested that all the jobs currently available are done, but they are working on more."

Reply:
Changed.